src/utils.js

```diff
@@ -84,7 +84,8 @@ export function debounce (fn) {
 export function getClosestIgnoredElement (element) {
   var parent = element;
 
-  while (parent && parent !== document && !(parent instanceof DocumentFragment)) {
+  // e.g. document doesn't have a function hasAttribute; no need to go further up
+  while (parent && typeof parent.hasAttribute === 'function' && !(parent instanceof DocumentFragment)) {
```

Comment:
Your future-safe when you check for function-existence. If you want to ensure that you reached top-level, you could also check validity of `parent.parentNode` instead of using `instanceof`.